tests/ListCommandTest.php

```diff
@@ -37,7 +37,13 @@ public function it_lists_hooks_that_exist()
     public function it_uses_a_different_git_path_if_specified()
     {
         $gitDir = 'test-git-dir';
-        passthru("mkdir -p {$gitDir}/hooks");
+
+        $command = "mkdir -p {$gitDir}/hooks";
```

Comment:
You know what, could we just use the [mkdir](php.net/mkdir) function? I should've done that in the first place. It would be much better.